features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^the following dishes exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:dish, hash)
+  end
+end
+
+When(/^I am on the landing page$/) do
+  visit dishes_path
+end
+
+Then(/^I should see "([^"]*)"$/) do |content|
+  expect(page).to have_content content
+end
```

Comment:
I always prefer to separate the step definition that are assertions into a separate file for better readability.